src/main/resources/resources/logisim/strings/std/std.properties

```diff
@@ -612,6 +612,8 @@ ramTypeAttr = Ram type
 ramTypeNonVolatile = non volatile
 ramTypeVolatile = volatile
 ramWithByteEnables = Use byte enables
+ramInvertOutputEnable = Invert output enable
+ramInvertWriteEnable = Invert write enable
```

Comment:
IMHO, it would be clearer for the users if we speak in case of enable, reset, and set signals of activity instead of negated/inverted inputs, i.e., the attribute refers to the control signal and the user has the choice between "high-active" and "low-active".

@figurantpp, @davidhutchens, @BFH-ktt1: What do you think?

Reply:
Something like "Output Enable Mode: High Active/Low Active" sounds ok to me

Reply:
I agree that it is clearer to state the active level rather than speak of inverting something. Active High/Active Low sounds better to me ear than High Active/Low Active.